sig-cli/charter.md

```diff
@@ -20,10 +99,16 @@ with Kubernetes API's. Examples of these binaries include: [kubectl and kustomiz
 
 ### Out of scope
 
-SIG CLI is not responsible for command-line tools built and maintained by other
-SIGs, such as kubeadm, which is owned by SIG Cluster Lifecycle. SIG CLI is not
-responsible for defining the Kubernetes API that it interfaces with. The
-Kubernetes API is the responsibility of SIG API Machinery.
+- SIG CLI is not responsible for command-line tools built and maintained by other
+  SIGs, such as kubeadm, (which is owned by SIG Cluster Lifecycle).
+- SIG CLI is not responsible for tools or solutions developed outside of the Kubernetes
+  project.
+- SIG CLI is not responsible for commands developed as plugins or other extension
```

Comment:
This may not be the case if existing built-in commands are converted to plugins, or plugins are used to prototype new built-in commands.

Reply:
That's why we say "outside of SIG CLI", even if built-in commands will turn into plugins (convert is one example since it relies on internal types that won't move with kubectl to a separate repo), but sig-cli will be still responsible for it and it will be developer under kubernetes org.

Reply:
If existing commands are converted to plugins how are they installed or how do people who used those commands in the past learn about how to get them back?